data/snippets/yaml.md

```diff
@@ -0,0 +1,28 @@
+```
+$ echo "---
+dict :
+  key:
+        value
+" >> test.yaml
+$ coala --files test.yaml --bears YAMLLintBear
```

Comment:
Sorry, but why? [Other examples](https://github.com/coala/landing-frontend/tree/a99aac1/data/snippets) don't use this option.